pkr/driver/docker_compose.py

```diff
@@ -160,13 +160,32 @@ def cmd_up(self, services=None, verbose=False, build_log=None):
         eff_modules = self._resolve_services(services)
 
         # Image names may be different from service names (e.g. image re-use)
-        images = set(
+        pattern = re.compile("^\S+\/\S+$")
```

Comment:
Can you add a comment to tell that this regex detects remote images

Reply:
Added